model/k8s/resources.yaml

```diff
@@ -111,3 +111,13 @@ groups:
     attributes:
       - ref: k8s.cronjob.uid
       - ref: k8s.cronjob.name
+
+  - id: resource.k8s.persistent_volume
+    type: resource
+    stability: experimental
+    name: k8s.persistent_volume
+    brief: >
+      A Kubernetes Persistent Volume object.
+    attributes:
```

Comment:
I guess this would at least need to define the `k8s.persistent_volume.name` and `k8s.persistent_volume.uid` attributes.